packages/calypso-products/src/constants/wpcom.ts

```diff
@@ -26,6 +26,7 @@ export const PLAN_BLOGGER_2_YEARS = 'blogger-bundle-2y';
 export const PLAN_ECOMMERCE_MONTHLY = 'ecommerce-bundle-monthly';
 export const PLAN_ECOMMERCE = 'ecommerce-bundle';
 export const PLAN_ECOMMERCE_2_YEARS = 'ecommerce-bundle-2y';
+export const PLAN_WOOEXPRESS_TRIAL_MONTHLY = 'ecommerce-trial-bundle-monthly';
```

Comment:
Let's keep this as `PLAN_ECOMMERCE_TRIAL_MONTHLY` - while we're building a WooExpress flow to offer the plan for now, the underlying plan is fairly generic in terms of what it represents.
```suggestion
export const PLAN_ECOMMERCE_TRIAL_MONTHLY = 'ecommerce-trial-bundle-monthly';
```